project_singlefamilydetached/resstock_cluster.json

```diff
@@ -1,12 +1,12 @@
 {
   "cluster_name": "resstock",
   "connected": false,
-  "server_instance_type": "c3.8xlarge",
+  "server_instance_type": "d2.4xlarge",
   "worker_instance_type": "c3.8xlarge",
   "user_id": "jrobert1",
-  "worker_node_number": 3,
+  "worker_node_number": 0,
   "aws_tags": [],
-  "openstudio_server_version": "2.8.1",
+  "openstudio_server_version": "2.9.0",
```

Comment:
Just want to confirm that this change was intentional.

Reply:
Yes, it should've already been at 2.9.0.